backend/src/main/resources/application.yaml

```diff
@@ -54,6 +54,9 @@ simple-report-initialization:
       manufacturer: LumiraDx UK Ltd.
       model: LumiraDx SARS-CoV-2 Ag Test*
       loinc-code: "95209-3"
+simple-report:
+  data-hub:
+    uploadurl: "https://prime-data-hub-test.azurefd.net/api/reports"
```

Comment:
`upload-url` seems more legible. (Kebab case is preferred for YAML—that'll turn into `uploadUrl` under certain property-management transforms)